src/overview/components/Filters.jsx

```diff
@@ -8,39 +8,79 @@ const Filters = ({
     onShowOnlyBookmarksChange,
     clearAllFilters,
     isClearFilterButtonShown,
+    tagFilterManager,
+    domainFilterManager,
+    onFilterClick,
+    setRef,
```

Comment:
should be a bit more descriptive - what ref is it setting? as there are multiple "sub-views" within this view (you could even separate these out into multiple views and reuse a lot of the structure - up to you)